### DISCUSSION TITLE
Landing Page

### DISCUSSION THREAD

**Discussion by mo-anon** [squidfunk/mkdocs-material#6638] [Q&A] [answered]
Im currently trying to create a cool landing page for my documentation.
Does anyone have any resources on how to create something similar to the current "Material for MK-Docs" page (it's soooo cool)?

I already tried to find the source code of it, but unfortunately it seems like its not public?
Greatful for every resource or guide on landing pages with mkdocs. Already saw https://github.com/squidfunk/mkdocs-material/issues/1996.

thanks,

**Comment by squidfunk** [comment, marked as answer]
Duplicate of https://github.com/squidfunk/mkdocs-material/discussions/4262 – source is available in [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/).
